lib/resolver/homebrew-file.js

```diff
@@ -2,7 +2,7 @@ import relativeFile from './relative-file.js';
 
 export default function ({ path, target }) {
   return [
-    relativeFile({ path, target }),
-    '{BASE_URL}/Homebrew/homebrew-core/blob/master/Formula/' + target,
+    relativeFile({ path, target, baseUrl: 'https://github.com' }),
```

Comment:
Yeah, everything looks good to me :)

I'll leave this and any other PRs alone so the release can go out. Let me know if you'd like any help with it.